checkpoint-management-console/playbooks/playbooks.json

```diff
@@ -0,0 +1,446 @@
+              "name": "Start",
+              "description": null,
+              "status": null,
+              "arguments": {
+                "step_variables": {
+                  "input": {
+                    "params": []
+                  }
+                }
+              },
+              "left": "20",
+              "top": "20",
+              "stepType": "/api/3/workflow_step_types/b348f017-9a94-471f-87f8-ce88b6a7ad62"
+            },
+            {
+              "uuid": "1f0ac430-c191-47d4-a984-517591ea2445",
+              "@type": "WorkflowStep",
+              "name": "Update Host",
+              "description": null,
+              "status": null,
+              "arguments": {
+                "name": "CheckPoint Management Console",
+                "config": "''",
+                "params": {
+                  "params_type": "UID",
+                  "interfaces": {
+                    "add": {
+                      "name": "Interface1",
+                      "subnet": "198.168.123.1",
+                      "mask-length": 24,
+                      "subnet-mask": "255.255.255.0",
+                      "color": "blue",
+                      "comments": "any comment",
+                      "details-level": "standard",
+                      "ignore-warnings": false,
+                      "ignore-errors": false
+                    }
+                  },
+                  "ip-address": "",
+                  "ipv4-address": "",
+                  "ipv6-address": "",
+                  "nat-settings": {
+                    "auto-rule": false,
+                    "ip-address": "198.168.123.1",
+                    "hide-behind": "gateway",
+                    "install-on": "",
+                    "method": "static"
+                  },
+                  "new-name": "",
+                  "tags": "",
+                  "host-servers": {
+                    "dns-server": true,
+                    "mail-server": true,
+                    "web-server": true,
+                    "web-server-config": {}
+                  },
+                  "color": "",
+                  "comments": "",
+                  "details-level": "standard",
+                  "groups": "",
+                  "ignore-warnings": "",
+                  "ignore-errors": ""
+                },
+                "version": "1.0.0",
+                "connector": "checkpoint-management-console",
+                "operation": "update_host",
+                "operationTitle": "Update Host"
+              },
+              "left": "188",
+              "top": "120",
+              "stepType": "/api/3/workflow_step_types/0bfed618-0316-11e7-93ae-92361f002671"
+            }
+          ],
+          "triggerLimit": null,
+          "description": "Update host based on the parameters that you have specified.",
+          "name": "Update Host",
+          "tag": "#CheckPoint Management Console",
+          "recordTags": [
+            "Checkpoint",
```

Comment:
P should be in capital letters . `CheckPoint`

Reply:
that is autogenerated tag so it should be fine

Reply:
done